feature/futuretodolist/src/main/java/com/phelat/tedu/futuretodolist/view/TodoListItem.kt

```diff
@@ -0,0 +1,50 @@
+package com.phelat.tedu.futuretodolist.view
+
+import android.graphics.Paint
+import androidx.core.content.ContextCompat
+import com.phelat.tedu.todo.entity.TodoEntity
+import com.phelat.tedu.futuretodolist.R
+import com.xwray.groupie.kotlinandroidextensions.GroupieViewHolder
+import com.xwray.groupie.kotlinandroidextensions.Item
+import kotlinx.android.synthetic.main.item_todo.todoTitle
+
+class TodoListItem(
```

Comment:
Dear @PHELAT , yes it would be definitely better , what do you think about todo module under common hierachy , is that good place or design system module ?

Reply:
Yeah, I think the `:common:todo` module would be a good place for this type of shared UI code.

Reply:
Done.